src/components/common/VFrame.vue

```diff
@@ -7,6 +7,16 @@
       @return="onReturn"
       @close="onClose"
     >
+      <div
```

Comment:
logo should be only for pages, where it required with this icon in design.
but we have some pages, who are used v-frame component, but no need to use this `logo`
please add special flag for show/hide logo

Reply:
hide by default